src/Public/Security.ps1

```diff
@@ -122,14 +128,40 @@ function Add-PodeSecurityHeader {
 
         [Parameter()]
         [string]
-        $Value
+        $Value,
+
+        [Parameter()]
+        [switch]
+        $Add,
+
+        [Parameter()]
+        [ValidateSet('Upper', 'Lower')]
+        [string]
+        $Casing
     )
 
     if (![string]::IsNullOrWhiteSpace($Value)) {
-        $PodeContext.Server.Security.Headers[$Name] = $Value
+        if ($add -and $PodeContext.Server.Security.Headers.ContainsKey($Name)) {
+            $Headers = $PodeContext.Server.Security.Headers[$Name].split(',')
+            if ($Headers -contains $Value) {
+                return
+            } else {
+                $Headers += $Value
+                $PodeContext.Server.Security.Headers[$Name] = (($Headers.trim() | Select-Object -Unique) -join ', ')
+            }
```

Comment:
I'd change this to `-inotcontains`, and swap the if/else around. Then drop the new else-block with just a `return` in it so it's just a simple if-statement.

Reply:
It was like that. Later I changed because I thought you preferred this way 😀😀😀😀